baseapp-wagtail/baseapp_wagtail/tests/utils/media_helper.py

```diff
@@ -0,0 +1,18 @@
+from io import BytesIO
+
+import PIL.Image
+from django.core.files.base import ContentFile
+from django.core.files.images import ImageFile
+
+
+def get_test_image_file(filename="test.png", colour="white", size=(640, 480)):

```

Comment:
There is a `image_djangofile` fixture in `baseapp_core.tests.fixtures` maybe we can reuse?

Reply:
I'm using them inside factories. Fixtures aren't designed to be added there. Besides, I'm using the `TestCase` class from the `unittest` package. Pytest fixtures aren’t natively compatible with `unittest`, which makes the code a bit messier. That said, I prefer to keep using `get_test_image_file` and `get_test_document_file`. What do you think?